src/unix/notbsd/linux/mod.rs

```diff
@@ -17,9 +17,14 @@ pub type msgqnum_t = ::c_ulong;
 pub type msglen_t = ::c_ulong;
 pub type nfds_t = ::c_ulong;
 pub type nl_item = ::c_int;
-
+pub type idtype_t = ::c_int;
 pub enum fpos64_t {} // TODO: fill this out with a struct
 
+pub const P_PID: ::id_t = 1;
+pub const P_PGID: ::id_t = 2;
+pub const P_ALL: ::id_t = 3;
```

Comment:
Update.

Also while I have your attention. How does one go about adding the `linux/*.h` sources for linking? 

I can see where to add them for the test suite/crate. This was the main issue why I couldn't merge the 2 remaining for `PTRACE` yesterday/last night.